data/test/scenarios/ability_id_active_test.cfg

```diff
@@ -1,3 +1,19 @@
+#####
+# API(s) being tested: [have_unit]ability_id_active
+##
+# Starting state:
+# Side 1 leader Alice (Elvish Archer)
+# Side 2 leader Bob (Orcish Grunt)
+##
+# Actions:
+# Give Alice an unconditionally active [hides] ability.
+# Recruit a Peasant and check that a unit exists with an active [hides] ability.
+# Recruit a Woodsman and check that a unit exists with an active [hides] ability.
```

Comment:
While you're not changing the test here, the test itself seems really weird - as if it's testing for an existing unit (Alice) having her data corrupted by the recruiting mechanism.

Reply:
Yeah I was wondering that myself.  It seems like it'd be an equivalent test to just add the ability and then immediately afterwards check whether the ability is active.